aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -181,7 +165,7 @@ IMPLEMENT_UNARY_OP_VEC(asin)
 IMPLEMENT_UNARY_OP_VEC(atan)
 IMPLEMENT_UNARY_OP_VEC(ceil)
 IMPLEMENT_UNARY_OP_VEC(cos)
-IMPLEMENT_UNARY_OP_TH(cosh)
+IMPLEMENT_UNARY_OP_VEC(cosh)
```

Comment:
Can we comment here, that kernels are not using Vec256 (because _VEC is confusing )